e2e-tests/README.md

```diff
@@ -7,54 +7,49 @@ Continuous Integration
 [![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.prod.svg?label=prod)](https://webqa-ci.mozilla.com/job/socorro.prod/)
 
 
-This directory holds Socorro client-based end-to-end tests which is why they're different than the rest of the code in this repository.
+This directory holds Socorro client-based end-to-end tests, which is why they're different than the rest of the code in this repository.
 
-To review the specific Python packages the tests utilize, please review `requirements.txt`.
+To review the specific-Python packages the tests use, please review `tox.ini`.
 
 Set up and run Socorro tests
 -----------------------------
 
 Review the documentation for [pytest-selenium][pytest-selenium] and decide which browser
 environment you wish to target.
 
-We suggest using a different virtual environment for these tests than the
-rest of Socorro so you're not mixing requirements:
+* [Install Tox](https://tox.readthedocs.io/en/latest/install.html)
+* Run `tox`
 
-	$ mkvirtualenv socorro-tests
-	$ # make sure you DON'T run the next command in
-	$ # your dev virtualenv.
-	$ pip install -r requirements.txt
+An additional constraint for Firefox users: with the release of Firefox 48, WebDriver support is currently broken until GeckoDriver is out of Alpha. We suggest using Firefox 47.0.2 which can be [downloaded here][firefoxdownloads].
 
-An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using Firefox 47.0.2 which can be [downloaded here][firefoxdownloads].
-
-If you have multiple versions of Firefox installed, you can specifiy a specific one by using the `--firefox-path <path to firefox binary>` flag.
+If you have multiple versions of Firefox installed, you can specify a specific one by using the `--firefox-path <path to firefox binary>` flag.
 
 ___Running the tests on stage___
 
-	$ py.test --driver Firefox tests/
+	$ tox
```

Comment:
This should be fine. These directions assume you're in the `e2e-tests/` directory when running these two commands.